### PR TITLE
[wallet] embed wallet.ini into wallet binary (#830)

### PR DESCRIPTION
**Pull request by touhonoob**
utilize go:generate to convert the default wallet.ini file into go constant `defaultWalletIni` defined in
`cmd/client/wallet/generated_ini.go` which will be used when `.hmy/wallet.ini` doesn't exist under the current filesystem.

## Issue

#830 

#### Test Coverage Data

* Before
https://gist.github.com/touhonoob/e8c4dd936bc06a408fd66906b2595091

* After
https://gist.github.com/touhonoob/ab3154577af32d219fb27641020519e3

#### Test/Run Logs

https://gist.github.com/touhonoob/c919ba36e926f22366fefdedf6af2806

### CODE REVIEW

**Comment by harmony-ek** [comment]
Jenkins speaking; can one of the admins verify this patch?

**Comment by LeoHChen** [comment]
Jenkins, test this please.

File: scripts/wallet_embed_ini_files.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+// Reads all .ini files in the `.hmy` folder of root path
+// then encodes them as strings literals in `cmd/client/wallet/generated_ini.go`
+func main() {
+	const root = "../../../"
+	hmy := path.Join(root, ".hmy")
+	fs, _ := ioutil.ReadDir(hmy)
+	out, _ := os.Create("generated_ini.go")
+	out.Write([]byte("package main\n\nconst (\n"))
+	for _, f := range fs {
+		if strings.HasSuffix(f.Name(), ".ini") {
```
Comment (LeoHChen) [inline scripts/wallet_embed_ini_files.go]: we have other .ini files, don't convert all of them.  maybe just wallet.ini

**Comment by touhonoob** [comment]
@LeoHChen addressed in https://github.com/harmony-one/harmony/pull/833/commits/9f73091edf672ca74b2f7397d96d1c8843921b1e

**Comment by LeoHChen** [comment]
Jenkins, test this please.

File: cmd/client/wallet/generated_wallet.ini.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package main
```
Comment (LeoHChen) [inline cmd/client/wallet/generated_wallet.ini.go]: do we need to check in the generated wallet.ini.go file?  Also, maybe we shall ignore it in .gitignore?

Comment (touhonoob) [reply cmd/client/wallet/generated_wallet.ini.go]: I believe so, otherwise the travis check script will fail.

**Pull request merged**